src/components/CategoryNavContainer/CategoryNav/CategoryNav.js

```diff
@@ -1,34 +1,62 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
 import style from './CategoryNav.scss';
 import { NO_CATEGORY } from '../../../constants';
 
-const CategoryNav = props => (
-  <div>
-    <NavLink
-      className={style.link}
-      activeClassName={style.link__active}
-      to={`/${props.cityId}/${NO_CATEGORY}`}
-    >Overview</NavLink>
+class CategoryNav extends Component {
+  componentDidMount() {
+    this.scrollToActiveNavLink();
+  }
 
-    {props.categories.map(category => (
-      <NavLink
-        className={style.link}
-        activeClassName={style.link__active}
-        key={category.id}
-        to={`/${props.cityId}/${category.id}`}
-      > {category.name} </NavLink>
-    ))}
-  </div>
-);
+  componentDidUpdate() {
+    this.scrollToActiveNavLink();
+  }
+
+  scrollToActiveNavLink() {
+    if (this.activeCategoryEl) {
+      this.el.scrollLeft = this.activeCategoryEl.parentElement.offsetLeft - 20;
```

Comment:
This has a bug on iOS where it selects two items:

![image](https://user-images.githubusercontent.com/4443482/31208444-b70cc5a4-a9d0-11e7-8b77-2ef369b6d301.png)

I think what happens is that the DOM updates while your finger is still on the screen. So I click on "Jobs & Skills" and it moves to the left, my finger is now over the "Housing" button, so iOS applies the hover/active style and for whatever reason after I lift up, it doesn't remove the hover style.

This fixes it:
```
if (this.activeCategoryEl) {
  setTimeout(() => {
    this.el.scrollLeft = this.activeCategoryEl.parentElement.offsetLeft - 20;
  });
}
```

When you do `setTimeout()` without the (optional) delay parameter it puts the callback function on the 'next tick'